src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -242,6 +242,78 @@ async function deserializeToCdpArg(
       return {objectId: argEvalResult.result.objectId};
     }
 
+    case 'PROTO.binding': {
+      // TODO(BLOCKER): Where do we use bindingName?
+      const bindingName = argumentValue.value;
+
+      const argEvalResult = await realm.cdpClient.sendCommand(
+        'Runtime.callFunctionOn',
+        {
+          functionDeclaration: String(() => {
+            const queue: string[] = [];
+            return {
+              /** Gets a promise, which is resolved as soon as a message occurs in the queue. */
+              async getMessage(): Promise<string> {
+                // long-poll
+                for (;;) {
+                  // TODO(BLOCKER): I am quite confused about this. How come bindingHandle is there? Is it referring to itself? Should it be "this" instead?
+                  // const message =
+                  await realm.cdpClient.sendCommand('Runtime.callFunctionOn', {
+                    functionDeclaration: String(() => {
+                      return this.getMessage();
+                    }),
+                    awaitPromise: true,
+                    generateWebDriverValue: true,
+                  });
+                  // TODO(BLOCKER): It's not obvious how to send an event. It seems we need eventManager, but none is available from scriptEvaluator.
+                  // No other cases in scriptEvaluator are currently sending any event.
+                  // sendBiDiEvent(
+                  //   'script.bindingCalled',
+                  //   serializeCdpObject(message.result, 'root', realm)
+                  // );
+                }
+              },
+              /** Adds a message to the queue. */
+              sendMessage(message: string) {
+                queue.push(message);
```

Comment:
It should add message to queue, and resolve Promise returned by `getMessage`

Reply:
Promise is only needed/used iff the queue is empty.

Reply:
`message.result` -> `{name: bindingName, value: message.result}`.